src/controller/CHIPDeviceController.h

```diff
@@ -191,6 +191,16 @@ class DLL_EXPORT DeviceController : public AbstractDnssdDiscoveryController
         return nullptr;
     }
 
+    Messaging::ExchangeManager * ExchangeMgr()
+    {
+        if (mSystemState != nullptr)
```

Comment:
if (mSystemState) to align with SessionMgr()

Reply:
@yufengwangca @yunhanw-google 
You two made the opposite request. Which is better?
https://github.com/project-chip/connectedhomeip/pull/29244#discussion_r1333742492

Reply:
I would prefer using nullptr makes it obvious that you are working with pointers for other people reading your code

Reply:
Both of them check if the mSystemState pointer is not null (or not a null pointer), the key is we should stick with one for consistence.

Both versions are commonly used, but the second one is more concise and is more used in our SDK due to its brevity and readability.